### ISSUE TITLE
Descending sort on ordinal scale

### ISSUE THREAD

**Issue by floribon** [vega/vega#515]
Hi there, it doesn't seem to be any way to sort the `bar` chart example of the editor by `y` values from highest to lowest.

We can sort them using the ordinal `x` scale domain:
`{"data": "table", "field": "x", "sort": {"field": "y", "op": "max"}}`

But how could we go descending instead of ascending? Using `-y` doesn't seem to work here. If that's a missing feature I think that's a pretty importat one. Otherwise that would be great to see it [in the doc](https://github.com/vega/vega/wiki/Scales#sorting-domains-for-ordinal-and-quantile-scales). Thanks!

TLDR how to get the opposite sorting of this chart:
<img width="556" alt="screen shot 2016-02-05 at 12 17 07 pm" src="https://cloud.githubusercontent.com/assets/2934743/12844876/73adce00-cc02-11e5-83d3-aa8b34e09ecc.png">

**Comment by arvind** [comment]
Hi @floribon, you can set `"reverse": true` on the ordinal scale to reverse the range like so: 

```json
 {
      "name": "x",
      "type": "ordinal",
      "range": "width",
      "domain": {"data": "table", "field": "x", "sort": {"field": "y", "op": "max"}},
      "reverse": true
    }
```

**Comment by floribon** [comment]
Of course! Thanks a lot again @arvind 

(It takes some practice to not be confused by the different ways to sort: using a transform, a scale domain, etc. Maybe a small article that sum things up in the future could help newcomers)

**Comment by arvind** [comment]
Agreed, it's on my todo list :)
